src/components/Navigation/Sidebar/LiveLeaderboard.js

```diff
@@ -0,0 +1,189 @@
+import { useMemo, useState, useEffect, useRef } from "react";
+import { ethers } from "ethers";
+import { useWeb3React } from "@web3-react/core";
+import Jazzicon, { jsNumberForAddress } from "react-jazzicon";
+import Davatar from "@davatar/react";
+import { useENS, truncateMiddleEthAddress, formatAmount, USD_DECIMALS } from "../../../Helpers";
+// import { ReactComponent as PositionIndicator } from "../../../img/position-indicator.svg";
+import * as Styles from "./LiveLeaderboard.styles";
+import liveIcon from "../../../img/nav/live.svg";
+import { useLocation } from "react-router-dom";
+
+const ARBISCAN_URL = "https://arbiscan.io/address/";
+// const MIN_UI_PERCENTAGE = 10;
+// const MAX_UI_PERCENTAGE = 91.3;
+const BOTTOM_PERCENT = 95;
+
+export const LiveLeaderboard = ({ leaderboardData, userPosition }) => {
+  const location = useLocation();
+  const { account } = useWeb3React();
+  const { ensName } = useENS(account);
+
+  const userPercentage = useMemo(
+    () => 100 - (userPosition / leaderboardData?.length) * 100,
+    [leaderboardData, userPosition]
+  );
+
+  const lastInTopFive = useMemo(() => {
+    if (!leaderboardData || !userPosition) {
+      return undefined;
+    }
+    const topFiveIndex = Math.floor(userPosition / leaderboardData?.length);
+    const topFivePercentUser = leaderboardData[topFiveIndex];
+    return topFivePercentUser;
+  }, [leaderboardData, userPosition]);
+
+  const differenceBetweenUserAndTopFive = useMemo(() => {
+    if (!leaderboardData || !userPosition || !lastInTopFive) {
+      return undefined;
+    }
+    const difference = ethers.BigNumber.from(lastInTopFive.volume).sub(
```

Comment:
would this need to have a `.add(1)` or something? This would be the volume required to be tied with the lowest person in top five but adding 1 would be the amount required to overtake them?

Reply:
I guess that is correct, but tying includes them both